internal/entities/command_deployment.go

```diff
@@ -41,14 +44,16 @@ The deployment command manages deployments for a New Relic entity. Use --help fo
 	Example: "newrelic entity deployment create --guid <GUID> --version <0.0.1>",
 }
 
+var cmdEntityDeploymentCreateExample = fmt.Sprintf(`newrelic entity deployment create --guid <GUID> --version <0.0.1> --changelog 'what changed' --commit '12345e' --customAttributes '{"region": "east", "env": "staging"}' --deepLink <link back to deployer> --deploymentType 'BASIC' --description 'about' --timestamp %v --user 'jenkins-bot'`, time.Now().Unix())
```

Comment:
Since we're hiding `--customAttributes` from the help output, we should also remove it from the example output.

Reply:
True, done in c25adb85f108e04194eabe5a48e98188b5d6d252